server/protocols/webadmin/README.adoc

```diff
@@ -0,0 +1,110 @@
+= Web administration for JAMES
+
+The web administration supports for now the CRUD operations on the domains and the users, as described in the following sections.
+
+**WARNING** : This API do not allow authentication for now. It means that an administrator should ensure an attacker can not use this API.
+
+== Administrating domains
+
+=== Create a domain
+
+.bash
+====
+curl -XPOST http://ip:port/domains/domainToBeCreated
```

Comment:
ok, for the third time : please describe domainToBeCreated. What is it ? Which contraints ? And it's still a PUT if you give the id.